app-text/htmltidy/tidy57-5.7.0~git.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="The granddaddy of HTML tools, with support for modern standards"
+DESCRIPTION="Tidy tidies HTML and XML. It can tidy your documents by itself, \
+and developers can easily integrate its features into even more powerful tools."
+HOMEPAGE="http://www.html-tidy.org/"
+COPYRIGHT="1998-2015 World Wide Web Consortium"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="28be79db50d2ecf6730068167d8ed23987fa8d59"
+SOURCE_URI="https://github.com/htacg/tidy-html5/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="5fddac299a385203e06a2158df3acd68aa83ffd10d5bdf73b9e7f2913dee1a88"
+SOURCE_FILENAME="tidy-html5-$portVersion.tar.gz"
+SOURCE_DIR="tidy-html5-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=5.7.0
+libVersionCompat="$libVersion compat >= 5"
+
+PROVIDES="
+	tidy57$secondaryArchSuffix = $portVersion
+	lib:libtidy$secondaryArchSuffix = $libVersionCompat
+	"
+if [ -z "$secondaryArchSuffix" ]; then
+PROVIDES="$PROVIDES
+	cmd:tidy = $portVersion
+	"
+fi
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+CONFLICTS="
+	tidy$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	tidy57${secondaryArchSuffix}_devel = $portVersion
+	devel:libtidy$secondaryArchSuffix = $portVersion
```

Comment:
Oops, I almost missed this one :)
```diff
-	devel:libtidy$secondaryArchSuffix = $portVersion
+	devel:libtidy$secondaryArchSuffix = $libVersion
```
Hmm, it was my fault! I made that mistake in an [old comment](https://github.com/haikuports/haikuports/pull/1932#discussion_r158699653). Sorry!